drivers/ethernet/eth_mcux.c

```diff
@@ -911,14 +911,13 @@ static struct device *eth_mcux_get_ptp_clock(struct device *dev)
 #endif
 
 static const struct ethernet_api api_funcs = {
-	.iface_api.init = eth_iface_init,
-	.iface_api.send = eth_tx,
-
-	.get_capabilities = eth_mcux_get_capabilities,
-
+	.iface_api.init		= eth_iface_init,
+	.iface_api.send		= net_eth_send,
```

Comment:
iface_api.send is never set by L2. Well here anyway it's temporary change since in the end that send pointer disappears altogether

Reply:
> iface_api.send is never set by L2. Well here anyway it's temporary change since in the end that send pointer disappears altogether

As the value is set to be the generic L2 send function, I was wondering if the L2 to set it automatically instead of each driver setting it. But if the field is going away, then just skip this question.